### ISSUE TITLE
Incorrect FPS with acceleration?

### ISSUE THREAD

**Issue by Yefancy** [SPICExLAB/MobilePoser#3]
https://github.com/SPICExLAB/MobilePoser/blob/main/mobileposer/process.py#L15
```python
TARGET_FPS = 30
```
according to your code, I noticed that you sampling the motion to `30hz`

however, the acceleration generation is `60Hz`
https://github.com/SPICExLAB/MobilePoser/blob/main/mobileposer/process.py#L23-L32
```python
def _syn_acc(v, smooth_n=4):
    """Synthesize accelerations from vertex positions."""
    mid = smooth_n // 2
    acc = torch.stack([(v[i] + v[i + 2] - 2 * v[i + 1]) * 3600 for i in range(0, v.shape[0] - 2)])
    acc = torch.cat((torch.zeros_like(acc[:1]), acc, torch.zeros_like(acc[:1])))
    if mid != 0:
        acc[smooth_n:-smooth_n] = torch.stack(
            [(v[i] + v[i + smooth_n * 2] - 2 * v[i + smooth_n]) * 3600 / smooth_n ** 2
             for i in range(0, v.shape[0] - smooth_n * 2)])
    return acc
```

I double check the transpose codebase, they were using the `60hz` both motion and acceleration by default.
https://github.com/Xinyu-Yi/TransPose/blob/main/preprocess.py

I'm not sure whether it will cause a bug?

**Comment by GaigeY** [comment]
Hi Ye, I am also working on validating the results of `MobilePoser` using my own motion capture dataset. I followed the AMASS processing script provided by the authors, similar to `IMUPoser`, to obtain the data for comparison.

However, I have encountered some similiar issues with the reconstruction results. I noticed that while you mentioned the results were not satisfactory, my findings have been quite similiar. I tested both 30Hz and 60Hz versions (modified with `TransPose` and `IMUPoser`), but both yielded high PA-MPJPE values exceeding 130 mm, whereas I observed lower values with TransPose (90+) and PIP (around 50+).

I am unsure if there might be an issue with my replication steps. Could you please share your replication results? I would greatly appreciate any insights you might have regarding your experience with MobilePoser.

Thank you for your time!

**Comment by sinchaoJa** [comment]
> Hi Ye, I am also working on validating the results of `MobilePoser` using my own motion capture dataset. I followed the AMASS processing script provided by the authors, similar to `IMUPoser`, to obtain the data for comparison.
> 
> However, I have encountered some similiar issues with the reconstruction results. I noticed that while you mentioned the results were not satisfactory, my findings have been quite similiar. I tested both 30Hz and 60Hz versions (modified with `TransPose` and `IMUPoser`), but both yielded high PA-MPJPE values exceeding 130 mm, whereas I observed lower values with TransPose (90+) and PIP (around 50+).
> 
> I am unsure if there might be an issue with my replication steps. Could you please share your replication results? I would greatly appreciate any insights you might have regarding your experience with MobilePoser.
> 
> Thank you for your time!


Hello, I used the `weights.pth` file provided by the author for evaluation. The experimental results suggest that the `weights.pth` file was most likely trained under 30Hz conditions, rather than 60Hz. This is my hypothesis.

**Comment by sinchaoJa** [comment]
> Hi Ye, I am also working on validating the results of `MobilePoser` using my own motion capture dataset. I followed the AMASS processing script provided by the authors, similar to `IMUPoser`, to obtain the data for comparison.
> 
> However, I have encountered some similiar issues with the reconstruction results. I noticed that while you mentioned the results were not satisfactory, my findings have been quite similiar. I tested both 30Hz and 60Hz versions (modified with `TransPose` and `IMUPoser`), but both yielded high PA-MPJPE values exceeding 130 mm, whereas I observed lower values with TransPose (90+) and PIP (around 50+).
> 
> I am unsure if there might be an issue with my replication steps. Could you please share your replication results? I would greatly appreciate any insights you might have regarding your experience with MobilePoser.
> 
> Thank you for your time!

I recently tested the dip_test.pt and imuposer_test.pt files generated using the process.py script provided by the author. With the weights.pth file and the ONLINE setup, I evaluated using the MPJVE metric. The results are as follows: `imuposer: 9.44, dip: 12.55`.
According to the author's paper, the fine-tuned results for imuposer should be `9.5` and for dip `12.2`.
I did not experiment with the TotalCapture dataset, as I am unclear about the author's data processing approach for this dataset.
Would you be willing to share your reproduction process with me?

**Comment by taeyoungyeon** [comment]
updated code for reflecting fps for calculation. Try again and whether issue was solved. Thanks
